### ISSUE TITLE
Selection toolbar didn't work after migrate null safe

### ISSUE THREAD

**Issue by theacheng** [singerdmx/flutter-quill#119]
There might be some code break after migration to null safe as I try to track it:
1. A commit before migrate:
```yaml
flutter_quill:
  git:
    url: https://github.com/singerdmx/flutter-quill.git
    ref: dda5805935568d42b71a1a1faf7fbdf078d72633
```

Selection toolbar work.
<img width="318" alt="image" src="https://user-images.githubusercontent.com/29684683/112725385-8c5a1d80-8f4a-11eb-8895-74cf722e8d7a.png">

2. A commit after migrate:
```yaml
flutter_quill:
  git:
    url: https://github.com/singerdmx/flutter-quill.git
    ref: 60514cbd24fbde3a4500ae04b930a039ee5851f4
```

Selection toolbar doesn't work.
<img width="318" alt="image" src="https://user-images.githubusercontent.com/29684683/112725610-b233f200-8f4b-11eb-83b8-3bcedba3f68f.png">

**Comment by singerdmx** [comment]
Are you able to track which line?

**Comment by singerdmx** [comment]
Is this web or mobile?

**Comment by theacheng** [comment]
It is on mobile. I tried to track it, but can't.

**Comment by singerdmx** [comment]
You can simple revert line/block by line/block

**Comment by singerdmx** [comment]
You already know 60514cbd24fbde3a4500ae04b930a039ee5851f4 works
Just apply later changes one by one
You will fine which one breaks

**Comment by singerdmx** [comment]
Actually dda5805935568d42b71a1a1faf7fbdf078d72633 works

**Comment by singerdmx** [comment]
check out https://github.com/singerdmx/flutter-quill/commit/ace4fd4deb929e37128f78252a9d5310dafe7d33

**Comment by friebetill** [comment]
The bug should be solved by #120, let me know if it worked for you.

**Comment by theacheng** [comment]
Working now, thank brother!

<img width="302" alt="image" src="https://user-images.githubusercontent.com/29684683/112731088-d69dc780-8f67-11eb-93ed-b581f6d5710d.png">
